static/css/components/home.less

```diff
@@ -70,6 +69,11 @@ div.chartHome a:hover {
   }
 }
 
+#home-stats-charts .statschart:hover a {
```

Comment:
This adds some additional CSS

I think in this file it makes sense to wrap the rule with 
```
/* stylelint-disable no-descending-specificity */ 
a:hover...
/* stylelint-enable no-descending-specificity */ 
```
and leave this file unchanged.